logistic_budget/tests/test_propagate_budget.py

```diff
@@ -19,12 +19,28 @@
 
 class TestPropagateBudget(TransactionCase):
     def test_propagate_budget_to_cost_estimate(self):
-        self.source.requisition_line_id.budget_tot_price = 120.0
         cel_data = self.wizard_model._prepare_cost_estimate_line(self.source)
         self.assertEqual(cel_data['budget_tot_price'], 120.0)
         for key in cel_data:
             self.assertIn(key, dir(self.env['sale.order.line']))
 
+    def test_propagate_budget_to_cost_estimate_2_sources(self):
+        source2 = Mock(
+            spec_set=self.env['logistic.requisition.source'],
+            dispatch_location_id=False,
+            procurement_method='other',
+            proposed_product_id=self.env['product.product'],
+            proposed_qty=80,
+            requisition_line_id=self.line,
+        )
+        self.line.requested_qty = 100
+        cel_data1 = self.wizard_model._prepare_cost_estimate_line(self.source)
+        cel_data2 = self.wizard_model._prepare_cost_estimate_line(source2)
+        self.assertEqual(cel_data1['budget_tot_price'] +
+                         cel_data2['budget_tot_price'], 120.0)
```

Comment:
I added some comments on what is 24 and 96. I preferred not to change former value of test.